drivers/staging/vc04_services/bcm2835-isp/Kconfig

```diff
@@ -0,0 +1,14 @@
+config VIDEO_ISP_BCM2835
+	tristate "BCM2835 ISP support"
+	depends on MEDIA_SUPPORT
+	depends on VIDEO_V4L2 && (ARCH_BCM2835 || COMPILE_TEST)
+	select BCM2835_VCHIQ_MMAL
+	select VIDEOBUF2_DMA_CONTIG
+	select V4L2_MEM2MEM_DEV
```

Comment:
Do we actually use anything from V4L2_MEM2MEM_DEV, or is this a leftover from bcm2835-codec?

Reply:
No, not needed any more.

Reply:
Actually don't we have a dependency on MEDIA_CONTROLLER here too?
I've lost track of which things are automatically selected and which aren't these days. Perhaps enough of MEDIA_CONTROLLER is stubbed when it isn't acutally built such that it works.

Reply:
I assumed MEDIA_SUPPORT pulled in the bits of media controller.

Seems to build correctly regardless.

Reply:
MEDIA_SUPPORT != MEDIA_CONTROLLER, it's any part of V4L2/DVB/Radio/SDR/etc.
It's defined in drivers/media/Kconfig, ie the base directory of all media.

It does appear that all the function prototypes in media-device.h are stubbed should CONFIG_MEDIA_CONTROLLER not be defined.
https://elixir.bootlin.com/linux/latest/source/include/media/media-device.h#L440

Reply:
Added MEDIA_CONTROLLER.